static/app/components/performance/teamKeyTransaction.tsx

```diff
@@ -232,6 +324,35 @@ function TeamKeyTransactionSelector({
   );
 }
 
+type ItemProps = {
+  team: Team;
+  isKeyed: boolean;
+  disabled: boolean;
+  onSelect: () => void;
+};
+
+function TeamKeyTransactionItem({team, isKeyed, disabled, onSelect}: ItemProps) {
+  return (
+    <DropdownMenuItem
+      key={team.slug}
+      disabled={disabled}
+      onSelect={onSelect}
+      stopPropagation
+    >
+      <MenuItemContent>
+        {team.name}
+        <ActionItem>
+          {disabled ? (
+            t('Max.%s', MAX_TEAM_KEY_TRANSACTIONS)
```

Comment:
unsure if this is a typo
```suggestion
            t('Max %s', MAX_TEAM_KEY_TRANSACTIONS)
```

Reply:
The `.` are from the designs, though I can see why you think it may be a typo. I'll remove them for now, and we can revisit if the UI is unclear.